### PR TITLE
SDPA-498 "Slugline" should be replaced by "Keyword"

### PR DESCRIPTION
**Pull request by mugurrus**
translation updates

### CODE REVIEW

**Comment by petrjasek** [comment]
:+1:

File: po/en_GB.po [hunk 1]
```diff
@@ -11,6 +11,10 @@ msgstr ""
 "Language: en_GB\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
+#: scripts/superdesk/lang/missing-translations-strings.js:19
+msgid "'SLUGLINE is a required field'"
+msgstr "'KEYWORD is a required field'"
```
Comment (petrjasek) [inline po/en_GB.po]: this probably should be empty like the others

Comment (mugurrus) [inline po/en_GB.po]: No, it's needed for PA

Comment (petrjasek) [inline po/en_GB.po]: that's strange, it's not there for `Slugline` below

**Comment by gbbr** [comment]
:+1:

**Comment by mugurrus** [comment]
This is not the proper solution to the issue, will make another PR.

**Pull request closed without merging**